src/controllers/user.controller.js

```diff
@@ -59,15 +59,45 @@ export default class UserController {
     const signinError = { message: 'Incorrect email or password' };
     User.findOne({ where: { email: req.body.email } })
       .then((user) => {
-        if (!user) return Response.UnauthorizedError(res, signinError);
+        if (!user) throw new Error();
         return UserUtils.validateUserPassword(user, req.body.password)
```

Comment:
Don't you think it would be better to have validations in the validations or middlewares folder? At least, it will make your code more readable.
Also, please use `async/await`  for consistency and improved readability

Reply:
This very validation is against the password, which means I've gone into the db already  and should I handle that in a middleware then I won't touch the DB in the controller (you recommended I call DB from controller the other time). 
 `async/await`, this is the local signin feature which got merged already, I only edited it to increase test coverage. The new feature `oauthSignin` implements `async/await`

Reply:
Well, that's okay.